dcap/quoteverify/src/quote_enclave.rs

```diff
@@ -412,32 +400,27 @@ mod test {
         assert_eq!(PathInitializer::ensure_initialized(), Ok(()));
     }
 
-    #[parameterized(
-    persistent = { RequestPolicy::Persistent },
-    ephemeral = { RequestPolicy::Ephemeral },
-    )]
-    #[serial]
-    fn load_policy_succeeds(policy: RequestPolicy) {
-        reset_load_policy_initializer();
-        assert_eq!(LoadPolicyInitializer::policy(policy), Ok(()));
-    }
-
     #[test]
-    #[serial]
-    fn load_policy_fails_when_already_initialized() {
-        reset_load_policy_initializer();
-        LoadPolicyInitializer::try_default().unwrap();
+    fn load_policy() {
```

Comment:
Probably for good reason, but a bit harder to test thuroughly the interface changed such that an error is returned if one tries to set the load policy more than once into the SDK.
See https://github.com/intel/SGXDataCenterAttestationPrimitives/releases/tag/DCAP_1.19
> Added a limitation in the DCAP QVL to only allow the user to set the QvE load policy once.